src/components/containers/container.ts

```diff
@@ -14,6 +14,7 @@ type ContainerType = {
     fd?: string
     wrap?: string
     fw?: string
+    scroll?: boolean
```

Comment:
same here 🙏

Reply:
Used in the Component: BreadcrumbsWrapper